userspace/libsinsp/metrics_collector.h

```diff
@@ -274,9 +272,11 @@ class libs_metrics_collector
 	uint32_t m_metrics_flags = METRICS_V2_KERNEL_COUNTERS | METRICS_V2_LIBBPF_STATS | METRICS_V2_RESOURCE_UTILIZATION | METRICS_V2_STATE_COUNTERS | METRICS_V2_PLUGINS;
 	std::vector<metrics_v2> m_metrics;
 
+#ifdef __linux__
```

Comment:
I think we might want to move these in the `scap_platform` vtable, likely as a 
`struct scap_metrics_vtable` (embedded in each `scap_foo_platform`), so that we could get platform-dependent metrics from the scap handle. Again, this might be an idea for a future refactor.